proparse/src/main/java/org/prorefactor/proparse/antlr4/StringFuncs.java

```diff
@@ -8,12 +8,23 @@
  * Contributors:
  *    John Green - initial API and implementation and/or initial documentation
  *******************************************************************************/ 
-package org.prorefactor.proparse;
+package org.prorefactor.proparse.antlr4;
 
 import java.util.HashSet;
 
 public class StringFuncs {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1118)